osquery/tables/system/darwin/os_version.cpp

```diff
@@ -37,6 +38,9 @@ QueryData genOSVersion(QueryContext& context) {
   }
 
   // The version path plist is parsed by the OS X tool: sw_vers.
+  // We set a special ENV to ensure despite the fact Osquery is built against
+  // an pre Big Sur SDK, we want the modern numbering scheme for macOS.
+  setEnvVar("SYSTEM_VERSION_COMPAT", "1");
```

Comment:
Shouldn't this be `0` ?

Reply:
Yep....interesting. I wonder if I actually need to build this on 10.15 to really make sure I did this right.